xbmc/video/VideoDatabase.cpp

```diff
@@ -2658,9 +2659,22 @@ bool CVideoDatabase::GetFileInfo(const std::string& strFilenameAndPath, CVideoIn
       return false;
 
     details.m_iFileId = m_pDS->fv("files.idFile").get_asInt();
-    details.m_strPath = m_pDS->fv("path.strPath").get_asString();
-    std::string strFileName = m_pDS->fv("files.strFilename").get_asString();
-    ConstructPath(details.m_strFileNameAndPath, details.m_strPath, strFileName);
+    // On STRM/M3U playlists that contains multiple media urls,
+    // each FileItem has the "path" and "filename path" that point to the playlist file
```

Comment:
Here, you do not have a `CFileItem`; `details` is a `CVideoinfoTag`. So, the comment is misleading.